src/pages/evm/allowances/AllowancesTable.vue

```diff
@@ -1,6 +1,110 @@
 <script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+
+import { ShapedAllowanceRow } from 'src/antelope/types/Allowances';
+import { useUserStore } from 'src/antelope';
+import { getCurrencySymbol } from 'src/antelope/stores/utils/currency-utils';
+
+import AllowancesTableRow from 'pages/evm/allowances/AllowancesTableRow.vue';
+import TableControls from 'components/evm/TableControls.vue';
+import { computed, ref } from 'vue';
+
+const props = defineProps<{
+    rows: ShapedAllowanceRow[];
+}>();
+
+const { t: $t } = useI18n();
+const { fiatLocale, fiatCurrency } = useUserStore();
+
+const fiatSymbol = getCurrencySymbol(fiatLocale, fiatCurrency);
+
+const pagination = ref({
+    page: 1,
+    rowsPerPage: 10,
+    rowsNumber: props.rows.length,
+});
+
+const tableColumns = [
+    {
+        name: 'asset',
+        field: 'asset',
+        label: $t('global.asset'),
+        align: 'left' as 'left',
+        sortable: true,
+    },
```

Comment:
I think you are missing the checkbox to select the line for later revoke.
(I don't know if this is the correct place)

Reply:
the checkboxes will be added in #605. they are omitted here because we want to have a "vertical slice" which is the View Allowances feature, #206, so this page actually adds value. the checkboxes will be added as a part of the revoke vertical slice, #207

Reply:
![image](https://github.com/telosnetwork/telos-wallet/assets/11837820/67cb5f9d-4497-48b8-b5e7-b95d657382d9)